app/pages/patientdata/patientdata.js

```diff
@@ -734,9 +734,15 @@ export let PatientData = React.createClass({
         this.deriveChartTypeFromLatestData(latestData, uploads)
       );
 
+      const datetime = _.get(this.props, 'queryParams.datetime');
+
       let state = {
         chartType,
-        initialDatetimeLocation: chartType === 'trends' ? latestData.time : null,
+        initialDatetimeLocation: datetime
```

Comment:
The double ternary gives me pause. Maybe it's best to move it to a nested if/else?

Reply:
Sure.  I can take care of this before it goes to staging.